controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -136,11 +149,122 @@ func (r *SriovNetworkNodePolicyReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 		return reconcile.Result{}, err
 	}
 
+	if err = r.syncOffloadMachineConfig(policyList); err != nil {
+		return reconcile.Result{}, err
+	}
+
 	// All was successful. Request that this be re-triggered after ResyncPeriod,
 	// so we can reconcile state again.
 	return reconcile.Result{RequeueAfter: ResyncPeriod}, nil
 }
 
+func (r *SriovNetworkNodePolicyReconciler) syncOffloadMachineConfig(pl *sriovnetworkv1.SriovNetworkNodePolicyList) error {
+	logger := r.Log.WithName("syncOffloadMachineConfig")
+	var err error
+	devList := []render.DeviceInfo{}
+	offload := false
+
+	for _, p := range pl.Items {
+		if p.Spec.EswitchMode != sriovnetworkv1.ESWITCHMODE_SWITCHDEV {
+			continue
+		}
+
+		for _, dev := range p.Spec.NicSelector.RootDevices {
```

Comment:
Since the default PF name is kept as it is after setting to switchdev mode. should this also include pfNames?